src/services/auth/auth.ts

```diff
@@ -0,0 +1,51 @@
+import URL from "../../utils/constants/url-constants.json";
+
+export const fetchToken = async (username: string, password: string) => {
+  const authUrl = `${import.meta.env.VITE_API_AUTH_URL}${URL.AUTH}`;
+
+  const response = await fetch(authUrl, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/x-www-form-urlencoded',
+    },
+    body: new URLSearchParams({
+      client_id: "hasura-app",
+      username,
+      password,
+      grant_type: "password",
+      client_secret: "9ca6e96d-f72e-4208-91f4-a2d8e681f767",
+    }),
+  });
```

Comment:
Why are we using `fetch`? Please use `axios` instead.

Reply:
I get a CORS error if I use Axios. I use fetch to avoid this.